GDJS/Runtime/runtimescene.ts

```diff
@@ -732,6 +737,8 @@ namespace gdjs {
      * Tool function filling _allInstancesList member with all the living object instances.
      */
     _constructListOfAllInstances() {
+      this._layerHighestZOrders.clear();
```

Comment:
is this required?

Reply:
Yes, if a layer has a highest z order (let's say 25) but at the next frame, no instance is on the layer, this value won't change whereas it should not be defined.